compiler/src/iree/compiler/Codegen/LLVMCPU/test/pipeline_disable_distribution_tests.mlir

```diff
@@ -0,0 +1,20 @@
+// RUN: iree-opt --pass-pipeline='builtin.module(iree-llvmcpu-select-lowering-strategy, func.func(iree-llvmcpu-lower-executable-target, iree-llvmcpu-check-ir-before-llvm-conversion))' --iree-llvmcpu-disable-distribution --split-input-file %s | FileCheck %s
+
+// Test that iree_linalg_ext.map_scatter op is not generated when distribution
+// is disabled.
```

Comment:
Presumably, `iree_linalg_ext.map_scatter` is something tied to distribution? This might be obvious to most 😅